pkgs/native_assets_cli/lib/src/api/ios_sdk.dart

```diff
@@ -2,17 +2,27 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import '../model/ios_sdk.dart' as model;
+part '../model/ios_sdk.dart';
 
 /// For an iOS target, a build is either done for the device or the simulator.
 ///
 /// Only fat binaries or xcframeworks can contain both targets.
```

Comment:
Could this sentence be expanded? I don't know what this means.

Reply:
Hm, maybe this is more confusing than explaining. A xcframework or fat binary basically contains more than one dynamic library. (So the multiple dylibs can target multiple architectures and multiple iOS sdks.) I'll remove it for now as users should not concern themselves with this right now. We take care of packaging dylibs in frameworks inside flutter_tools.